src/newgrf_debug_gui.cpp

```diff
@@ -1025,6 +1032,13 @@ struct SpriteAlignerWindow : Window {
 				this->SetWidgetLoweredState(widget, SpriteAlignerWindow::crosshair);
 				this->SetDirty();
 				break;
+
+			default:
+				if (IsInsideBS(widget, WID_SA_ZOOM, ZOOM_LVL_END)) {
```

Comment:
```suggestion
				if (IsInsideBS(widget, WID_SA_ZOOM, ZOOM_LVL_COUNT)) {
```
For some reason the enumeration having both end and count, seems to imply that begin might not be at 0. But then, the trick with adding the enumeration to the base widget might fail when begin is non-zero.